### PR TITLE
Merge monitoring infogathering jumpscripts with alerts one

### PR DESCRIPTION
**Pull request by abdulrahmantkhalifa**
All alerts are now infogathering scripts.
raised correct eco warning/errors.
the new files are the ones in alerts u can compare to them

### CODE REVIEW

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_db.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 organization = "jumpscale"
 name = 'info_gather_db'
 author = "zains@codescalers.com"
+category = "monitor.healthcheck"
```
Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_db.py]: Since this job doesnt implement the monitor.healthcheck interface we should not add this category. is used by healthcheck though

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+def action():
+    results =list()
+    for ays in j.atyourservice.findTemplates():
+        instances = ays.listInstances()
+        for instance in instances:
+            aysinstance = ays.getInstance(instance)
+            result = dict()
+            result['state'] = 'OK'
+            result['message'] = "Process %s:%s:%s is running" % (aysinstance.domain, aysinstance.name, instance)
+            result['category'] = 'CPU'
```
Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py]: Category should be the ame as in line 38

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            if not aysinstance.actions.check_up_local(aysinstance, wait=False):
+                message = "Process %s:%s:%s is not running" % (aysinstance.domain, aysinstance.name, instance)
+                j.errorconditionhandler.raiseOperationalWarning(message, 'monitoring')
+                result['state'] = 'WARNING'
+                result['message'] = message
+                result['category'] = 'Proccess'
```
Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py]: Category should be same as in line 32
I suggest you call it AYS Process

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_redis.py [hunk 4]
```diff
@@ -37,23 +37,28 @@ def action():
...
                 if (used_memory / maxmemory) * 100 > 90:
                     state = 'WARNING'
-
-            size, unit = j.tools.units.bytes.converToBestUnit(used_memory)
-            msize, munit = j.tools.units.bytes.converToBestUnit(maxmemory)
-            used_memory = '%.2f %sB' % (size, unit)
-            maxmemory = '%.2f %sB' % (msize, munit)
-            result['message'] = '*Port*: %s. *Memory usage*: %s/ %s' % (port, used_memory, maxmemory)
+                    j.errorconditionhandler.raiseOperationalWarning(result['message'], 'monitoring')
+
```
Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_redis.py]: Why did you remove the unit stuff?

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_db.py (hunk 1 above)
Comment (abdulrahmantkhalifa) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_db.py]: so just remove the category or change  it to healthcheck

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    results =list()
+    for ays in j.atyourservice.findServices():
+        if not ays.getProcessDicts():
+                continue
+        result = dict()
+        result['state'] = 'OK'
+        result['message'] = "Process %s:%s:%s is running" % (ays.domain, ays.name, ays.instance)
+        result['category'] = 'AYS Process'
+        if not ays.actions.check_up_local(ays, wait=False):
+            message = "Process %s:%s:%s is not running" % (ays.domain, ays.name, ays.instance)
```
Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py]: improve this message

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            j.errorconditionhandler.raiseOperationalWarning(message, 'monitoring')
+            result['state'] = 'WARNING'
```
Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py]: this should be halted

File: apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py (hunk 5 above)
Comment (abdulrahmantkhalifa) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py]: "Process %s: %s: %s ---> is running"
"Process %s: %s: %s ---> Halted "

do you want extra info in the message

Comment (grimpy) [inline apps/agentcontroller/jumpscripts/core/monitoring_infogathering/info_gather_jsprocess_status.py]: Thats even worse Process %s: %s: %s will be fine running/halted is not needed since thats basicly the status.
